front/js/components/order.es6

```diff
@@ -217,6 +216,8 @@
         .then(formData => renderYandexForm(formData))
         .then(() => submit(DOM.yandexForm));
     } else {
+      // onOrderSend must be triggered before submit
+      mediator.publish('onOrderSend');
```

Comment:
you reproduced the old logic here. But seems it was wrong: we should publish `onOrderSend` in isYandex case too.
Do it here or create pdd subtask plz

Reply:
I am going to create a todo, because we should test it